### ISSUE TITLE
[BUG] Omni-modal inference python3 -m align_anything.serve.omni_modal_cli --model_name_or_path openbmb/MiniCPM-o-2_6 not work！

### ISSUE THREAD

**Issue by tensorflowt** [PKU-Alignment/align-anything#128]
### Required prerequisites

- [x] I have read the documentation <https://align-anything.readthedocs.io>.
- [x] I have searched the [Issue Tracker](https://github.com/PKU-Alignment/align-anything/issues) and [Discussions](https://github.com/PKU-Alignment/align-anything/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [x] Consider asking first in a [Discussion](https://github.com/PKU-Alignment/align-anything/discussions/new).

### What version of align-anything are you using?

0.0.1.dev0 /align-anything/align-anything-main

### System information

When I called the following inference method as instructed, I encountered an error：
`python3 -m align_anything.serve.omni_modal_cli --model_name_or_path openbmb/MiniCPM-o-2_6`
My model download source：https://www.modelscope.cn/models/OpenBMB/MiniCPM-o-2_6

`Package                           Version       Editable project location
--------------------------------- ------------- -----------------------------------
absl-py                           2.1.0
accelerate                        1.3.0
aiofiles                          23.2.1
aiohappyeyeballs                  2.4.4
aiohttp                           3.11.12
aiohttp-cors                      0.7.0
aiosignal                         1.3.2
airportsdata                      20241001
align-anything                    0.0.1.dev0    /align-anything/align-anything-main
altair                            5.5.0
annotated-types                   0.7.0
anyio                             4.8.0
args                              0.1.0
astor                             0.8.1
attrs                             25.1.0
audioread                         3.0.1
av                                14.1.0
blake3                            1.0.4
blinker                           1.9.0
braceexpand                       0.1.7
cachetools                        5.5.1
certifi                           2025.1.31
cffi                              1.17.1
charset-normalizer                3.4.1
click                             8.1.8
clint                             0.5.1
clip                              0.2.0
cloudpickle                       3.1.1
colorful                          0.5.6
compressed-tensors                0.9.0
datasets                          3.2.0
decorator                         5.1.1
deepspeed                         0.16.3
depyf                             0.18.0
diffusers                         0.32.2
dill                              0.3.8
diskcache                         5.6.3
distlib                           0.3.9
distro                            1.9.0
docker-pycreds                    0.4.0
einops                            0.8.0
fairscale                         0.4.13
fastapi                           0.115.8
ffmpy                             0.5.0
filelock                          3.17.0
flash_attn                        2.7.4.post1
frechet-audio-distance            0.1.2
frozenlist                        1.5.0
fsspec                            2024.6.1
ftfy                              6.3.1
gguf                              0.10.0
gitdb                             4.0.12
GitPython                         3.1.44
google-api-core                   2.24.1
google-auth                       2.38.0
googleapis-common-protos          1.66.0
gradio                            5.15.0
gradio_client                     1.7.0
grpcio                            1.70.0
h11                               0.14.0
hf_transfer                       0.1.9
hjson                             3.1.0
hpsv2                             1.2.0
httpcore                          1.0.7
httptools                         0.6.4
httpx                             0.28.1
huggingface-hub                   0.28.1
idna                              3.10
image-reward                      1.5
imageio                           2.37.0
imageio-ffmpeg                    0.6.0
importlib_metadata                8.6.1
iniconfig                         2.0.0
interegular                       0.3.3
Jinja2                            3.1.4
jiter                             0.8.2
joblib                            1.4.2
jsonschema                        4.23.0
jsonschema-specifications         2024.10.1
lark                              1.2.2
lazy_loader                       0.4
librosa                           0.10.2.post1
llvmlite                          0.44.0
lm-format-enforcer                0.10.9
Markdown                          3.7
markdown-it-py                    3.0.0
MarkupSafe                        2.1.5
mdurl                             0.1.2
mistral_common                    1.5.2
modelscope                        1.22.3
moviepy                           2.1.2
mpmath                            1.3.0
msgpack                           1.1.0
msgspec                           0.19.0
multidict                         6.1.0
multiprocess                      0.70.16
narwhals                          1.25.2
nest-asyncio                      1.6.0
networkx                          3.3
ninja                             1.11.1.3
numba                             0.61.0
numpy                             1.26.4
nvidia-cublas-cu12                12.4.5.8
nvidia-cuda-cupti-cu12            12.4.127
nvidia-cuda-nvrtc-cu12            12.4.127
nvidia-cuda-runtime-cu12          12.4.127
nvidia-cudnn-cu12                 9.1.0.70
nvidia-cufft-cu12                 11.2.1.3
nvidia-curand-cu12                10.3.5.147
nvidia-cusolver-cu12              11.6.1.9
nvidia-cusparse-cu12              12.3.1.170
nvidia-ml-py                      12.570.86
nvidia-nccl-cu12                  2.21.5
nvidia-nvjitlink-cu12             12.4.127
nvidia-nvtx-cu12                  12.4.127
openai                            1.61.1
opencensus                        0.11.4
opencensus-context                0.1.3
opencv-python                     4.6.0.66
opencv-python-headless            4.11.0.86
orjson                            3.10.15
outlines                          0.1.11
outlines_core                     0.1.26
packaging                         24.2
pandas                            2.2.3
partial-json-parser               0.2.1.1.post5
peft                              0.14.0
pillow                            10.4.0
pip                               25.0
platformdirs                      4.3.6
pluggy                            1.5.0
pooch                             1.8.2
proglog                           0.1.10
prometheus_client                 0.21.1
prometheus-fastapi-instrumentator 7.0.2
propcache                         0.2.1
proto-plus                        1.26.0
protobuf                          3.20.3
psutil                            6.1.1
py-cpuinfo                        9.0.0
py-spy                            0.4.0
pyarrow                           19.0.0
pyasn1                            0.6.1
pyasn1_modules                    0.4.1
pybind11                          2.13.6
pycountry                         24.6.1
pycparser                         2.22
pydantic                          2.10.6
pydantic_core                     2.27.2
pydeck                            0.9.1
pydub                             0.25.1
Pygments                          2.19.1
pytest                            7.2.0
pytest-split                      0.8.0
python-dateutil                   2.9.0.post0
python-dotenv                     1.0.1
python-multipart                  0.0.20
pytorch-fid                       0.3.0
pytz                              2025.1
PyYAML                            6.0.2
pyzmq                             26.2.1
ray                               2.42.0
referencing                       0.36.2
regex                             2024.11.6
requests                          2.32.3
resampy                           0.4.3
rich                              13.9.4
rpds-py                           0.22.3
rsa                               4.9
ruff                              0.9.4
safehttpx                         0.1.6
safetensors                       0.5.2
scikit-learn                      1.6.1
scipy                             1.15.1
semantic-version                  2.10.0
sentencepiece                     0.2.0
sentry-sdk                        2.20.0
setproctitle                      1.3.4
setuptools                        75.8.0
shellingham                       1.5.4
six                               1.17.0
smart-open                        7.1.0
smmap                             5.0.2
sniffio                           1.3.1
soundfile                         0.13.1
soxr                              0.5.0.post1
starlette                         0.45.3
streamlit                         1.42.0
sympy                             1.13.1
tenacity                          9.0.0
tensorboard                       2.18.0
tensorboard-data-server           0.7.2
threadpoolctl                     3.5.0
tiktoken                          0.7.0
timm                              1.0.13
tokenizers                        0.21.0
toml                              0.10.2
tomlkit                           0.13.2
torch                             2.5.1+cu124
torchaudio                        2.5.1+cu124
torchlibrosa                      0.1.0
torchvision                       0.20.1+cu124
tornado                           6.4.2
tqdm                              4.67.1
transformers                      4.48.2
triton                            3.1.0
typer                             0.15.1
typing_extensions                 4.12.2
tzdata                            2025.1
urllib3                           2.3.0
uvicorn                           0.34.0
uvloop                            0.21.0
virtualenv                        20.29.1
vllm                              0.7.1
wandb                             0.19.6
watchdog                          6.0.0
watchfiles                        1.0.4
wcwidth                           0.2.13
webdataset                        0.2.100
websockets                        14.2
Werkzeug                          3.1.3
wheel                             0.45.1
wrapt                             1.17.2
xformers                          0.0.28.post3
xgrammar                          0.1.11
xxhash                            3.5.0
yarl                              1.18.3
yt-dlp                            2025.1.26
zipp                              3.21.0
`

### Problem description

![Image](https://github.com/user-attachments/assets/7215adf0-6ea4-4897-8279-ed835b2e9acc)

### Reproducible example code

none

### Traceback

```pytb

```

**Comment by XuyaoWang** [comment]
From your error message, it seems that there is an issue with your network, preventing the successful download of model-related files from Hugging Face. You can try downloading [MiniCPM-o-2_6](https://huggingface.co/openbmb/MiniCPM-o-2_6) to your local machine first, and then use the command:
```
python3 -m align_anything.serve.omni_modal_cli --model_name_or_path <Your local path of MiniCPM-o-2_6>
```
to start Gradio. I just tested specifying the local path, and it started up normally.
